client/src/components/LoginModal/LoginModal.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Fragment } from 'react';
+import { Button, Modal } from 'react-bootstrap';
+
+const LoginModal = () => {
+  const [show, setShow] = useState(false);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+
+  return (
+    <Fragment>
```

Comment:
https://reactjs.org/docs/fragments.html#short-syntax

Reply:
This, somehow hasn't been working for me. In this setup. Maybe I am doing something wrong. I have pushed this for refactoring in the end.